content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -169,130 +169,158 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
     kube-proxy        2         2         2       2            2           beta.kubernetes.io/os=linux                                 26d
     ```
 
+
+
 #### Join Windows Worker
 
 In this section we'll cover configuring a Windows node from scratch to join a cluster on-prem. If your cluster is on a cloud you'll likely want to follow the cloud specific guides in the next section.
 
 #### Preparing a Windows Node
 
 {{< note >}}
-All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Admin).
+All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Administrator) on the Windows worker node.
 {{< /note >}}
 
-1. Install Docker (requires a system reboot)
+1. Download the [SIG Windows tools](https://github.com/kubernetes-sigs/sig-windows-tools) repository containing install and join scripts
+   ```PowerShell
+   [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+   Start-BitsTransfer https://github.com/kubernetes-sigs/sig-windows-tools/archive/master.zip
+   tar -xvf .\master.zip --strip-components 3 sig-windows-tools-master/kubeadm/v1.15.0/*
+   Remove-Item .\master.zip
+   ```
 
-    Kubernetes uses [Docker](https://www.docker.com/) as its container engine, so we need to install it. You can follow the [official Docs instructions](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon#install-docker), the [Docker instructions](https://store.docker.com/editions/enterprise/docker-ee-server-windows), or try the following *recommended* steps:
+1. Customize the Kubernetes [configuration file](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/Kubeclustervxlan.json) 
 
-    ```PowerShell
-    Enable-WindowsOptionalFeature -FeatureName Containers
-    Restart-Computer -Force
-    Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
-    Install-Package -Name Docker -ProviderName DockerMsftProvider
     ```
```

Comment:
Suggestion: use a `codenew` Hugo shortcode, and place this in a separate file

Reply:
Thanks for the suggestion - This is showing up as a pointer to the old instructions, which chunk of code are you referring to?